client/src/Components/SettingsPage/SettingsPage.js

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom';
+import stylesheet from './SettingsPage.css';
+
+const SettingsPage = () => {
+  return (
+    <>
+      <Link to="/"><div className='black-shade'></div></Link>
+      <div className='settings-container'>
```

Comment:
settings-container div should be top level tag of Settings component.

Reply:
For rendering reasons, our implementation wouldn't work with this. The settings page and the clickable background need to be on the same level for the user to be able to click to go back to the game container.